packages/catalog-search/package.json

```diff
@@ -66,6 +66,7 @@
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-instantsearch-dom": "^6.8.3",
+    "react-responsive": "^8.2.0",
```

Comment:
`MediaQuery` is a pass-thru component from `@edx/paragon` already, so we shouldn't need to explicitly install `react-responsive` here.

Reply:
fwiw, the linter did insist that I install it in `dependencies`:
```  4:24  error  'react-responsive' should be listed in the project's dependencies. Run 'npm i -S react-responsive' to add it  import/no-extraneous-dependencies```